app/javascript/app/document-capture/components/acuant-capture.jsx

```diff
@@ -29,6 +30,7 @@ function AcuantCapture() {
         <FullScreen onRequestClose={() => setIsCapturing(false)}>
           <AcuantCaptureCanvas
             onImageCaptureSuccess={(nextCapture) => {
+              onChange({ selfie: nextCapture.image.data });
```

Comment:
This is largely intended as placeholder behavior, just so that the endpoint responds successfully. #4009 includes a more finalized approach, where `AcuantCapture` manages `value` and `onChange` as a [controlled input](https://reactjs.org/docs/forms.html#controlled-components).